.github/workflows/e2e-test.yaml

```diff
@@ -0,0 +1,131 @@
+name: e2e-testpmd
+run-name: e2e-testpmd initiated by ${{ github.actor }}
+on:
+  pull_request:
+    types: [ labeled ]
+  workflow_dispatch:
+    inputs:
+      trex:
+        description: 'TREX version'
+        required: true
+        default: '2.88'
+        type: choice
+        options:
+        - '2.88'
+        - '3.02'
```

Comment:
Since we hard code the PCI addresses for the trafficgen ICE devices, I now wonder if we actually need this trex version option or simply fix it to 2.88, as this is the only version that works with the ICE physical NIC, 3.02 does not work with the ICE PF, but work with the VF.

Reply:
Removed.